elements/pfe-icon/src/pfe-icon.scss

```diff
@@ -107,13 +107,23 @@ $sizing: (
 // @TODO: Sort out sizing
 @each $label, $size in $sizing {
   :host([size="#{$label}"]) {
-    --pfe-icon--size: #{$size};
+    // Respect var if user set CSS var, but fallback to sizing property map
+    $desired-size: pfe-local(size, $fallback: $size);
+    max-width:   calc(#{$desired-size} + #{pfe-local(Padding)} * 2);
```

Comment:
This looks really great, I love the addition of the $desired-color and $desired-size to create that shared source of truth.  Can you tell me more about the addition of padding into the width calculation?

Reply:
I'm wondering also if, without breaking it out into `PaddingTop`, `PaddingRight` etc. if this will be an issue.  Imagine I've done: `--pfe-icon--Padding: 10px 5px;` in my stylesheet.  How would that work inside this calc?

Reply:
I'm just maintaining what was happening when the variables were updated (which turned out to be much more complicated than I expected). If you look here: https://github.com/patternfly/patternfly-elements/blob/b8a3c6e8ba0e5e97286c9765fee9159f51bd15f0/elements/pfe-icon/src/pfe-icon.scss#L40

That's what I'm maintaining, I think it's because the width is set to the SVG, and the max-width/height are set on the host, so the host has to take the padding into account. I thought about messing with it to see if I could get `box-sizing` or something a little more straightforward to make the same result, but I'm thinking it wouldn't work and I was having enough issues updating the SCSS and not getting a regression, so I just maintained what was there.

Reply:
Awesome!  Thanks for clarifying.  I'll add an issue to re-address that later.

Reply:
https://github.com/patternfly/patternfly-elements/issues/1502